src/go/rpk/pkg/cli/redpanda/admin/config/config.go

```diff
@@ -70,10 +74,81 @@ func newLogLevelCommand(fs afero.Fs, p *config.Params) *cobra.Command {
 	}
 	cmd.AddCommand(
 		newLogLevelSetCommand(fs, p),
+		newLogLevelGetCommand(fs, p),
 	)
 	return cmd
 }
 
+func newLogLevelGetCommand(fs afero.Fs, p *config.Params) *cobra.Command {
+	var host string
+
+	cmd := &cobra.Command{
+		Use:   "get [LOGGERS...]",
+		Short: "Get broker logger's log level",
+		Long: `Get broker logger's log level.
+
+This command obtains the broker logger's log level. Each Redpanda
+broker has many loggers, and each can be individually returned.
+
+You can specify many loggers at once by separating each logger name with a space.
+To return all loggers, you can leave the logger argument empty or pass "all". 
+To see all possible loggers, run the following command:
+
+  redpanda --help-loggers
+`,
+
+		Example: `  # List log levels for raft and kafka
+  rpk redpanda admin config log-level get kafka raft --host localhost:9644
+`,
+
+		Run: func(cmd *cobra.Command, loggers []string) {
+			p, err := p.LoadVirtualProfile(fs)
+			out.MaybeDie(err, "rpk unable to load config: %v", err)
+
+			cl, err := adminapi.NewHostClient(fs, p, host)
+			out.MaybeDie(err, "unable to initialize admin client: %v", err)
+
```

Comment:
Lets add `config.CheckExitCloudAdmin(p)` to exit early if a user tries to run this command against a Redpanda Cloud cluster.